walkthroughs/week-1-web-development/portfolio-walkthrough.md

```diff
@@ -164,10 +164,10 @@ several files and directories:
 -   <walkthrough-editor-open-file
         filePath="software-product-sprint/walkthroughs/week-1-web-development/examples/stanley/pom.xml">
       pom.xml
-    </walkthrough-editor-open-file>
+    </walkthrough-editor-open-file>cd
```

Comment:
This seems like an unintentional change?

Reply:
Yes, this was unintentional. Thank you for pointing it out.